src/CMakeLists.txt

```diff
@@ -325,7 +325,7 @@ endif()
 # are already loaded) and probably fail unless we set up LD_LIBRARY_PATH.
 if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
   # import library created by the `leanshared` target
-  set(LEANC_SHARED_LINKER_FLAGS "${LEANC_SHARED_LINKER_FLAGS} $bindir/libleanshared.dll.a")
+  set(LEANC_SHARED_LINKER_FLAGS "${LEANC_SHARED_LINKER_FLAGS} -lleanshared")
```

Comment:
why cant this library dependency be a cmake target DEPENDS ?  is it because "leanshared" is not a cmake "add_library" somewhere? If so, why isn't it?

Reply:
This flag is baked into `leanc`, i.e. user programs are linked with it as well

Reply:
I see thanks.

Reply:
By the way the title of this PR confuses me, "testing & distribution", isn't stdlib.make a central part of the regular build too?  What does "distribution" mean?

Reply:
Distribution means what we ship to users, i.e. the release tarballs created by `cpack`